x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -798,3 +930,24 @@ async function injectExtraFindData(
     referencedByCount: aggregationResult.aggregations[actionResult.id].doc_count,
   }));
 }
+
+function parseDate(dateString: string | undefined, propertyName: string, defaultValue: Date): Date {
```

Comment:
nit: i would probably move this to `iso_or_relative_date.ts` and then rename that file since it has more functions than just the `parseIsoOrRelativeDate` function

Reply:
Resolved in this commit 578d4334020d003439299ed65f05994a1ecdf4c7